run_tests.py

```diff
@@ -0,0 +1,18 @@
+import os
+import subprocess
+
+import nose
+
+
+root = os.path.abspath(os.path.join(__file__, ".."))
+
+# Setup environment for executable
+os.environ["PATH"] += os.pathsep + os.path.join(root, "volume")
+
+# Build docker file
+subprocess.call(["docker", "build", ".", "-t", "getavalon/docker"], cwd=root)
```

Comment:
Maybe the instructions to run the tests, needs to include the building step so running the tests do not actually build anything?